src/index.js

```diff
@@ -377,8 +376,7 @@ class MiniCssExtractPlugin {
                         '}',
                       ])
                     : '',
-                  'var head = document.getElementsByTagName("head")[0];',
-                  'head.appendChild(linkTag);',
+                  'document.head.appendChild(linkTag);',
```

Comment:
Hm IE9 doesn't support `getElementsByTagName`?

Reply:
@evilebottnawi looks safe to me?

https://caniuse.com/#search=getElementsByTagName

Reply:
@ScriptedAlchemy for me too, so i asked question why it was changed

Reply:
Yeah good point. Though both of them do work with old I.E. 

However I’m unsure how widely supported ‘document.head’ is especially in older browsers

Reply:
`document.head` is supported in IE9+.
https://developer.mozilla.org/en-US/docs/Web/API/Document/head#Browser_compatibility

Reply:
Looks like Webpack core has also switched over to using `document.head`:
https://github.com/webpack/webpack/pull/8467